### ISSUE TITLE
Bumping version in app.json to package.json version

### ISSUE THREAD

**Issue by SystemDisrupt** [expo-community/standard-version-expo#4]
### Description of the bug
When dry running standard-version for native application, it tries to bump app.json version, android and ios version to the version in package.json

### To Reproduce
What steps did you perform which led to this issue?
`npm i -D standard-version@next standard-version-expo`

.versionrc.js
`module.exports = {
  bumpFiles: [
    {
      filename: 'package.json',
    },
    {
      filename: 'app.json',
      updater: require.resolve('standard-version-expo'),
    },
    {
      filename: 'app.json',
      updater: require.resolve('standard-version-expo/android'),
    },
    {
      filename: 'app.json',
      updater: require.resolve('standard-version-expo/ios'),
    }
  ],
  releaseCommitMessageFormat: "chore(release): Release {{currentTag}}\n\n[ci skip]"
};`

I also tried 
`
...
      updater: require.resolve('standard-version-expo/android/increment'),
...
      updater: require.resolve('standard-version-expo/ios/ios-increment'),
...
`
Documentation was wrong for ios resolver. Is /increment, but should be /ios-increment for v1.0.0

app.json
`{
...
    "version": "2.2.1",
...
    "ios": {
...
      "buildNumber": "1",
...
    },
...
    "android": {
...
      "versionCode": 20033,
...
    },
...
}`

#### Expected behavior
- app.json version to be equal to package.json version 
- ios buildNumber to be a single integer 
- android versionCode to be in the format of 20033, but incremented

#### Actual behavior
Output of `npx standard-version --dry-run`
✔ bumping version in package.json from 1.24.0 to 1.25.0
✔ bumping version in app.json from 2.2.1 to 1.25.0
✔ bumping version in app.json from 20033 to 1.25.0
✔ bumping version in app.json from 1 to 1.25.0
✔ outputting changes to CHANGELOG.md

**Comment by byCedric** [comment]
Thanks for the report! The ios resolvers should work at `standard-version-expo/ios/increment`, I'll push a non-breaking fix for that asap. If I understand correctly, the dry run is also bumping versions for you? I need to take a closer look at that one, which might be an issue with standard version itself.

**Comment by byCedric** [comment]
- As for the same version in `package.json` and `app.json`, it uses the `package.json` as a start version. So if that's different compared to your `app.json` version, it will use the previous one. Maybe I should have documented that better.

- The dry run mode is not quite visible, it tells you that all the steps are done right? I can't seem to reproduce it in such a way that it still modifies your files and/or tags. Did that command change anything for you in the repo?

**Comment by SystemDisrupt** [comment]
That's cool, that's what I would expect to happen. It takes 1.24.0 from my package.json and bumps it to 1.25.0 in package.json and app.json.
 
The problem is that it's not calculating the `expo.ios.buildNumber` or `expo.android.versionCode` in the app.json correctly. It sets them to 1.25.0 also, which isn't the valid format for either. They should both be whole integers. 

Correct, the dry run is NOT modifying any files and just outputting what the standard command would do.

**Comment by byCedric** [comment]
Thanks for the additional info, I'll try to reproduce it and report back asap! 😁

**Comment by SystemDisrupt** [comment]
After further investigation it turns out the output from the `--dry-run` command is wrong, but it is actually changing the version and build numbers correctly. 
For some reason the commit wasn't registering on my local machine but is there when pulling after my CircleCI build. 

I'd also used the `standard-version-expo/ios` updater initially which changed my build number to 1.25.0. I then used `standard-version-expo/ios/increment` which of course tried to increment 1.25.0. 
I've now opted to used `standard-version-expo/ios/code` and `standard-version-expo/android/code` for consistency.

Thank you for fixing the ios aliases and apologies for reporting a false bug. The bug is in fact with `standard-version --dry-run` when processing the output from `standard-version-expo`
